.github/workflows/label-prs.yml

```diff
@@ -0,0 +1,26 @@
+name: 'Pull Request Labeler'
+on:
+  - pull_request_target
+
+jobs:
+  labeler:
+    name: 'Tag PRs by code changes'
+    permissions:
+      contents: read
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/labeler@v5
+        with:
+          sync-labels: true
```

Comment:
Related to the comment above: if a human manually adds a label, this means the bot will remove it? I don't know that we want to surrender control like this. Am I thinking about this wrong? If the bot gets a label wrong, would the correct response be to update the rules?

Reply:
So for titles, I'm fine leaving the sync off, so that humans can fix auto-tagged mistakes.

Reply:
I made this change to not sync titles in a4bd8a93abb40e178ebcc001af8e3443898f1c82.

Reply:
Help me understand what will happen during a sync. Say on a previous run, the title labeler added the "foo" label. Now we sync, and `actions/labeler` sees "foo" and that doesn't match any of the labels we'd otherwise apply. Will it then remove that label? In other words, will the two actions fight over whether or not a label should be applied?

Reply:
Since `actions/labeler` only adds tags, and this labeler would only run at the PR creation time, I don't think this is an issue.

Reply:
Resolved since sync is now False for both.